test/WebSites/VersioningWebSite/VersionAttribute.cs

```diff
@@ -32,7 +33,12 @@ public int Order
 
         public bool IsReusable => true;
 
-        public IActionConstraint CreateInstance(IServiceProvider services)
+        IActionConstraint IActionConstraintFactory.CreateInstance(IServiceProvider services)
+        {
+            return new VersionRangeValidator(_minVersion, _maxVersion) { Order = _order ?? 0 };
+        }
+
+        IEndpointConstraint IEndpointConstraintFactory.CreateInstance(IServiceProvider services)
```

Comment:
I'm not asking you to change it, but I do want to point out that it's wierd this uses a factory

Reply:
That was my thoughts exactly. My theory is it is testing two birds with one stone.